### PR TITLE
[READY] LabsSec™; Lightly Buffs Security, Security Armor Overhaul

### PR DESCRIPTION
**Pull request by ShadeAware**
## About The Pull Request
This PR is centered around buffing the defensive power of Security, and their ability to survive one-on-one encounters with dangerous events or antagonists.
Security armor and arms have had a small overhaul; standard Security Armor has been largely replaced by the **Security Techarmor**. This suit takes time to put on, but covers the entire body and comes with a built-in helmet with flash-resistant SecuHUD visor and Headlamp.
Security Disablers have been somewhat replaced with the new **Energy Rifle**, a type of energy gun which is far more bulky and cannot be put into backpacks. It does, however, sport a slightly longer charge, able to fire 25 (+5) disabler shots and 12 high energy blasts. It does not have a taser function. **The lethal functionality is locked to alert levels BLUE and higher, attempting to set it to lethal on Green Alert will cause the rifle to automatically swap back.**
yes
newsec
based
image
(Default, ERT reskins - Leftside shows the normal Security Techarmor, rightside shows the HoS Techarmor)
image
(Classic reskin variant)

## Why It's Good For The Game
Security is, for all intents and purposes, extremely underpowered in most situations where they have not either emptied the armory of its contents or powergamed to some capacity. Armor in its current state does little to assist them against the more dangerous hazards such as Antagonists, which can often lead to entire swaths of Security exercising poor decisions out of fear and the apparent need to do a "first strike".
This PR aims to help, if nothing else, give Security a much needed step up whilst not making them blatantly overpowered. By allowing Security to feel powerful, a large chunk of the fear and first-strike unga mentality can finally be put to rest and allow them to think more carefully and critically, knowing they are less likely to get two-tapped by one of the many _le funny syndicate items_, and that they're able to put up an actual fight.
Additionally, moving away from the standard Retro-Futurism artstyle into a more futuristic and cyberpunk-lite esq tone will greatly help oust the normal TG/Citadel LRP-MRP mindset, and serve as a big step forward into creating and maintaining both our own identity and a more MRP-HRP atmosphere.

## Changelog
:cl:
balance: Security Armor has been largely replaced by a new Security Techarmor, sporting full body coverage and a slight +5 boost to melee defense alongside a built in helmet with flash resistant HUD Visor and headlamp. This suit has an equip delay similar to hardsuits. CREDIT TO JUNGLERAT FOR SPRITES.
balance: Security Disablers have been largely replaced by the new Energy Rifle, a far more bulky energy gun which cannot be put into backpacks but sports a longer cell lifespan, able to fire 25 disabler shots and 12 lethal blasts.
add: Energy Rifle.
add: Security Techarmor. CREDIT TO JUNGLERAT FOR SPRITES.
add: Head of Security Techarmor.
add: Energy Rifle and Security Techarmor cargo crates; Armory Access is required to open them.
tweak: Replaced standard Security Armor spawns with the new Security Techarmor; Energy Rifles added to every map armory.
/:cl:

### CODE REVIEW

**Comment by Yawet330** [comment]
speedmerge

**Comment by Investigator77** [comment]
GET TACTICAL MARINES

**Comment by BobJoga** [comment]
One step closer to eris

**Comment by CameronWoof** [comment]
These would be cool options for the armory, but security rolling around in baby hardsuits with lethal eguns at roundstart seems excessive. It also, sad to say, just doesn't look as good as their current uniform. Maybe you could move the armor vests to the sec vendors so there's still a way to get the old-style stuff if you like it?

**Comment by ShadeAware** [comment]
> 
> 
> These would be cool options for the armory, but security rolling around in baby hardsuits with lethal eguns at roundstart seems excessive. It also, sad to say, just doesn't look as good as their current uniform. Maybe you could move the armor vests to the sec vendors so there's still a way to get the old-style stuff if you like it?

The suit doesn't have any space protection and the lethal functionality is explained in the **why its good for the game** section.
Old armor vests can certainly be added to the Security Vendor; I shall do that right now.

File: code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -316,3 +316,39 @@
...
 	armor = list("melee" = 25, "bullet" = 20, "laser" = 20, "energy" = 10, "bomb" = 20, "bio" = 50, "rad" = 20, "fire" = -10, "acid" = 50)
+
+//New and improved Tech Armor
+/obj/item/clothing/head/helmet/space/hardsuit/security_armor
+	name = "security exoskeleton helmet"
+	desc = "A specialized exoskeleton armor helmet built into a suit of armor; offers decent protection, and comes with a flash-resistant HUD visor and headlamp."
+	icon_state = "hardsuit0-secexo"
+	item_state = "hardsuit0-secexo"
+	hardsuit_type = "secexo"
+	armor = list("melee" = 40, "bullet" = 30, "laser" = 30,"energy" = 10, "bomb" = 25, "bio" = 0, "rad" = 0, "fire" = 55, "acid" = 70)
```
Comment (KathrinBailey) [inline code/modules/clothing/suits/armor.dm:327]: Why is the helmet armour block different from the suit armour block?

Comment (ShadeAware) [reply code/modules/clothing/suits/armor.dm:327]: The stats are copy/pasted from the normal security helmets
image

**Comment by ThatGuyFromThatPlace** [comment]
I feel like, security shouldn't wear these outside of a code emergency (Blue+, literally any code except green), but, that's just me - The person who doesn't wear their helmet unless it's code blue or higher

And, personally, I very much enjoy the taser mode, so, I'd hope hybrid tasers are still accessible. I generally use it in a combo of 1-2 disabler shots to slow, taser to freeze their movement, couple more disabler shots to stam-crit. I find the taser is, good for, not take-down speed, but, keeping the target still; Just dumping disabler shots is outright more effective than tasing for speed, but, for non-violent runners, the taser is, extremely useful

**Comment by YakumoChen** [comment]
Definitely would prefer to have the helmet down when you're not fighting, as having the whole of the sec force look like faceless corpsec thugs, while a nice thought for the cyberpunk theme, doesn't look good when you're trying to be casual.

**Comment by ShadeAware** [comment]
> 
> 
> Definitely would prefer to have the helmet down when you're not fighting, as having the whole of the sec force look like faceless corpsec thugs, while a nice thought for the cyberpunk theme, doesn't look good when you're trying to be casual.

The helmet can be retracted on both suit types.

**Comment by Tupinambis** [comment]
I think from both an artistic and roleplay perspective that the design of the suits, while certainly looking nice, would not fit in with security's standard attire. This is space security, not a space SWAT or space marine team, and having security look and feel like hyper-lethal vectors probably isn't good for roleplay or for encouraging "ideal behavior" when it comes to antags. It also looks like it SHOULD be space proof, which I guarantee will lead to confusion. There is a very good reason this sort of armor is relegated to the ERT, since the ERT is meant to be more of a proper PMC or SWAT force.

**Comment by Owai-Seek** [comment]
Just going to drop these here for comparison.

Riot Suit (Locked in the armory.)

Helmet
`("melee" = 45, "bullet" = 15, "laser" = 5,"energy" = 5, "bomb" = 5, "bio" = 2, "rad" = 0, "fire" = 50, "acid" = 50)`
Suit
`("melee" = 50, "bullet" = 10, "laser" = 10, "energy" = 10, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 80, "acid" = 80)`
Other Notes:
Blocks Shove Knockdown, Not Spaceworthy.

Swat Suit (Cargo order.)

Helmet
`("melee" = 40, "bullet" = 30, "laser" = 30,"energy" = 30, "bomb" = 50, "bio" = 90, "rad" = 20, "fire" = 50, "acid" = 50)`
Suit
`("melee" = 40, "bullet" = 30, "laser" = 30,"energy" = 30, "bomb" = 50, "bio" = 90, "rad" = 20, "fire" = 100, "acid" = 100)`
Other Notes:
Helmet is spaceworthy, suit is not.

Security Exosuit (This PR)

Helmet
`("melee" = 40, "bullet" = 30, "laser" = 30,"energy" = 10, "bomb" = 25, "bio" = 50, "rad" = 0, "fire" = 55, "acid" = 70)`
Suit
`("melee" = 35, "bullet" = 30, "laser" = 30, "energy" = 10, "bomb" = 25, "bio" = 50, "rad" = 0, "fire" = 55, "acid" = 70)`
Other Notes: In-Built Gasmask. THICKMATERIAL, so it blocks epipens, hyposprays, ect. Not spaceworthy. In-Built Flashlight

Aside from explosions and disabler shots, this PR effectively gives every security officer (and anyone who breaks into a security locker) access to a suit similar in defense to a swat suit, roundstart.

**Comment by ShadeAware** [comment]
> 
> 
> Just going to drop these here for comparison.
> 
> Riot Suit (Locked in the armory.)
> 
> Helmet
> `("melee" = 45, "bullet" = 15, "laser" = 5,"energy" = 5, "bomb" = 5, "bio" = 2, "rad" = 0, "fire" = 50, "acid" = 50)`
> Suit
> `("melee" = 50, "bullet" = 10, "laser" = 10, "energy" = 10, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 80, "acid" = 80)`
> Other Notes:
> `Blocks Shove Knockdown, Not Spaceworthy.`
> 
> Swat Suit (Cargo order.)
> 
> Helmet
> `("melee" = 40, "bullet" = 30, "laser" = 30,"energy" = 30, "bomb" = 50, "bio" = 90, "rad" = 20, "fire" = 50, "acid" = 50)`
> Suit
> `("melee" = 40, "bullet" = 30, "laser" = 30,"energy" = 30, "bomb" = 50, "bio" = 90, "rad" = 20, "fire" = 100, "acid" = 100)`
> Other Notes:
> `Helmet is spaceworthy, suit is not.`
> 
> Security Exosuit (This PR)
> 
> Helmet
> `("melee" = 40, "bullet" = 30, "laser" = 30,"energy" = 10, "bomb" = 25, "bio" = 50, "rad" = 0, "fire" = 55, "acid" = 70)`
> Suit
> `("melee" = 35, "bullet" = 30, "laser" = 30, "energy" = 10, "bomb" = 25, "bio" = 50, "rad" = 0, "fire" = 55, "acid" = 70)`
> `Other Notes: In-Built Gasmask. THICKMATERIAL, so it blocks epipens, hyposprays, ect. Not spaceworthy. In-Built Flashlight`
> 
> Aside from explosions and disabler shots, this PR effectively gives every security officer (and anyone who breaks into a security locker) access to a suit similar in defense to a swat suit, roundstart. The suit also gives them immunity fro gas-based attacks, removes the need for internal masks, and gives them a flashlight.

### That is an absolute baldface lie on the Swat Suit. All stats besides fire and acid were directly copied from their normal vest counterparts besides, with melee given a small +5 boost. If you are not going to stop lying to try and get the PR denied, I ask you please stop commenting on the PR at all.

**Comment by Owai-Seek** [comment]
I'm not lying at all and I have the stats right there. I said similar to, not an exact copy. For everything that (usually) matters (IE: Melee, Laser, and Bullet) damage, it's the same.

There's differences for Energy, Bomb, and Fire damage, which are the second most common damage types, which I -also- highlighted/mentioned. (But forgot to mention when I rewrote my message, which is why I posted the stats as well.)

I'm not going to stop commenting about balance concerns if there's actual balance concerns to address. I'm not trying to close the PR, just take things into consideration.

**Comment by YakumoChen** [comment]
Make sure the regular sec helmets, armor, and tasers are available for roboticists to build ED209

**Comment by CameronWoof** [comment]
Now that this is more fully-formed, I'll give my opinions all at once.

From a purely subjective aesthetic view, I kinda don't like this. The suits are all over the place, don't conform to any one specific theme or style (different visor colors, different palettes, etc.), and also seem to be trying too hard to be ERT cosplay. 

Beyond that, though, I feel it's disingenuous to describe this as a "light" buff to Security. With these proposed changes, Security is going from head, chest, and groin armor roundstart to _full body protection_, in addition to the actual numbers tweaks, as well as an all-in-one helmet that is simultaneously a pair of HUD sunglasses and a gasmask. They are also shifting from an exclusively nonlethal loadout to a lethal-capable one, which in itself has a big impact on both their engagement potential and the perception of the department in roleplay.

I'm a big proponent of policy reform aimed at giving Security a fighting chance, but I can't see any long-term positives coming out of a shift like this. This feels like the first move in an escalating arms race between security and antagonists, who now are much more justified in hunting Security officers with lethal intent due to their increased danger _and_ the greatly increased rewards.

Security's roundstart armor is intended to be markedly worse than the improved options in the armory, in order to give the escalation a definite curve and pattern, but this PR blurs that line a lot. 

And as a closing note, why do these hide tauric features? That's not something that any actual hardsuits do, and it seems out of place here.

**Comment by ShadeAware** [comment]
> From a purely subjective aesthetic view, I kinda don't like this. The suits are all over the place, don't conform to any one specific theme or style (different visor colors, different palettes, etc.), and also seem to be trying too hard to be ERT cosplay.

Kind of too bad, considering new spritework was added that invalidates this point entirely. The ERT look was a codersprite that has been shifted to a reskin look instead of the default.

> Beyond that, though, I feel it's disingenuous to describe this as a "light" buff to Security. With these proposed changes, Security is going from head, chest, and groin armor roundstart to _full body protection_, in addition to the actual numbers tweaks, as well as an all-in-one helmet that is simultaneously a pair of HUD sunglasses and a gasmask. They are also shifting from an exclusively nonlethal loadout to a lethal-capable one, which in itself has a big impact on both their engagement potential and the perception of the department in roleplay.

Previous armor was absolutely useless. Entirely. Why? Because the armor only protected the head and chest; Areas of which nobody actually targets. Because of this, roundstart security gear was **the exact same as having absolutely no armor at all.** All it provided was a place to store your gun; This is not good game design. This is not good balance. This is an error of which has needed a correction for far too long.
Furthermore, the lethal option is provided for the _off chance it is required._ Lethal firing for no reason is an in-character issue that would likely lead to said officer being fired; the actual lethal option is being provided because _most if not all Antagonists have some form of easy-to-get device or ability that completely invalidates Security stamina weapons, making it impossible for them to defend themselves from certain death._

> I'm a big proponent of policy reform aimed at giving Security a fighting chance, but I can't see any long-term positives coming out of a shift like this. This feels like the first move in an escalating arms race between security and antagonists, who now are much more justified in hunting Security officers with lethal intent due to their increased danger _and_ the greatly increased rewards.

Antagonists have never been allowed to hunt Security for no reason or a reason that amounts to "loot, epic nae nae". This point holds no water, and would fit more if this were a Citadel PR. But its not.

> Security's roundstart armor is intended to be markedly worse than the improved options in the armory, in order to give the escalation a definite curve and pattern, but this PR blurs that line a lot.

Worse does not mean **completely useless in every single facet its supposed to offer some form of advantage in.**

> And as a closing note, why do these hide tauric features? That's not something that any actual hardsuits do, and it seems out of place here.

Normal Hardsuits do hide tauric features; the tags had to be adjusted from default to not hide anything but said features (Jumpsuit, boots, etc).

> EDIT: And not to put too fine a point on it, but I'm not following the logic that this sprite change is going to have any meaningful effect on the level of roleplay present on the server. That doesn't seem like a very good argument in favor of this PR.

If things look shitty or weak, people are going to assume they are shitty and weak even when they shouldn't be. Security looked more like a bunch of people in Halloween costumes than they did a private security force hired to actually protect you.

**Comment by SiliconMain** [comment]
please make a seperate PR for the baton sprite change, its gonna be controversial

**Comment by Owai-Seek** [comment]
100% disagree with the baton changes, they make it much harder to tell someone has a stun baton out, and the detective already has a police baton.

**Comment by YakumoChen** [comment]
:-1: to stun baton sprites. It just looks like a black stick inhands. Stun batons have an extremely unique and obvious look which, in a game utilizing extremely low graphics, is a must have to identify it from 10,000 other items with an item as important to combat meta as the stun baton.

**Comment by Echopsychotic** [comment]
Why did you buff Bio, fire, and acid?

**Comment by ShadeAware** [comment]
It's almost like this is a fucking buff PR or something.

**Comment by Echopsychotic** [comment]
But like this is supposed to be the sec armor, but it covers all limbs. Let it be a sec armor. Cut the other crap. let it be exactly what it is supposed to be. sec armor that covers all limbs. Don't change the other stuff.

**Comment by Commandersand** [comment]
i think they're right about the batons, you should make it have a higher contrast color line one side or something so that it's better to differentiate between the detective's 

i think the more janky mall cop security vests/helmets fits ss13 more for casual security armor but the new armor looks epic.
i think you can make them more COOL than the current armor without it looking like a spacesuit by
making the helmet not be a full helmet, like just crop off the bottom and have it just be the visor+helmet top+light, i think the gas mask portion is unnecessary 
i think having visible armor chinks would help make it look less like a spacesuit while having full body protection making sense, like doom eternals armor.
image

is the far right armor supposed to be a new hardsuit armor? if so i think he head looks too round? it's not too important but i think having a more weirdly shaped head for security hardsuits would make them stand out more

**Comment by YakumoChen** [comment]
image
for real though, which of these in-hands are you more likely to think "oh he's got a stun baton"?

**Comment by Seris02** [comment]
Sec officers should _**not**_ be starting off with this.
###  Even if it isn't a fucking riot suit, it **_looks_** like one. 
Antags are going to take one look and be like:

# "Nope."

And it's just weird to be walking around in full body armor on a green shift.
If there's a threat, then they should put it on. Not get put in it during shift start.

**Comment by XyvMeph** [comment]
The guns literally violate rule 6.
All this should be what they switch to as soon as blue gets announced. None of it is reasonable on a green shift when the station basically *is* a mall.

**Comment by IrkallaEpsilon** [comment]
Baton sprites are a bit hard to read when inhand.
The roundstart energy gun should be in the armory, perhaps replace it with a disabler rifle or something, there are less than lethal longarms IRL, perhaps have a carbine that basicly just has the disabler (and maybe the inefficient tazer function). The armor sprites needs fixing for anything that isnt human, the helmet does not properly deploy and turns you into a bald headed human. Please dont add these generic vulpine/canine sprites we have for none humans to them.

I am glad they get full limb protection. I am also very glad for the "deliberately incompetent look" being gone with the new armors. I hate the entire mallcop stigma and that they are perceived as deliberately incompetent.

Limb protection is important for minor shit like spiders, vines etc. So you dont get dabbed on by legit anything due to bobmed. Most traitor/antag gear still has intrinsic armor pen so they can still cause wounds (Esword, Epen, Armblade, Pistol, Revolver just to name a few). The armor will not increase their surviveability by large as some say. The armor also does nothing against stam drain/stuns. The gun however will cause issues. 

Gas protection can be achieved with the old sec by simply going into your internal box and wearing that muzzle (I dont bother calling that a gasmask, it is not.). I dont think it will protect from the touch component of gas attacks but then again I legitimately have never seen a traitor use gas grenades (like the ones using phlogiston. Got sec on your ass? Use that.. really.. Its the best deterrent).

Thing has some issues but the idea is overall good in my personal opinion. 

I really HATE mallcopsec. This makes them not look like deliberately incompetent/perceived as incompetent mallcops.

**Comment by H0gberg** [comment]
I really like certain aspects of this PR, mainly the full body coverage, I think it's a wonderful step forward as well as the armor sprites themselves, however I think the energy rifle and baton sprite change should be put in different PR's for some more focused conversation as well possibly removing the +5 melee buff cause it just seems overall a bit unneeded.

**Comment by H0gberg** [comment]
also if you could fix the bald thing that would go a long way for me personally!

**Comment by ShadeAware** [comment]
### The Energy Rifle will not be changed or removed for reason outlined in the PR proper.

**Comment by Gandalf2k15** [comment]
In my opinion this change is much needed. Will check later. +1 here.

**Comment by Echopsychotic** [comment]
So this PR will break rules. So nice to know that you will not follow them because you want sec to have their powergame gear early.

**Comment by Commandersand** [comment]
huh i thought i commented this

oh well, one more thing. i'm not sure if it's intended but the full body armor blocks syringes/medibots/sleepers so it's more of a pain to heal security especially if they're dead/unconscious since you have to manually take off their helmets

**Comment by ShadeAware** [comment]
> 
> 
> huh i thought i commented this
> 
> oh well, one more thing. i'm not sure if it's intended but the full body armor blocks syringes/medibots/sleepers so it's more of a pain to heal security especially if they're dead/unconscious since you have to manually take off their helmets

Intended

**Comment by ShadeAware** [comment]
> 
> 
> The guns literally violate rule 6.
> All this should be what they switch to as soon as blue gets announced. None of it is reasonable on a green shift when the station basically _is_ a mall.

>Weak laser pretty much only used for simplemobs and emergency life-or-death situations
>Powergame

Im not even going to bother with a long winded response, just that thats a pretty ridiculous statement and not something im going to take seriously.

**Comment by KathrinBailey** [comment]
image
??????

**Comment by ShadeAware** [comment]
> 
> 
> image
> ??????

Actual differences in the belt icons to show if its off or doesn't have a cell is a bad thing now, is it?

**Comment by CameronWoof** [comment]
I like the alert-locked lethal function, but it is far too easy to subvert (the ability to subvert it with screwdriver > multitool or emag swiping should also be in the PR description). The emag function is fine, but "smart" officers hacking their gear roundstart is going to become a powergame issue really quickly.

**Comment by ThatGuyFromThatPlace** [comment]
> Sec officers should _**not**_ be starting off with this.
> 
> ### Even if it isn't a fucking riot suit, it **_looks_** like one.
> Antags are going to take one look and be like:
> 
> # "Nope."
> And it's just weird to be walking around in full body armor on a green shift.
> If there's a threat, then they should put it on. Not get put in it during shift start.

Security, as it is, has zero defense against the legs or arms. The arms are particularly vulnerable because breaking them makes people drop weapons.
This is a station where nearly every day there are: Traitors, changelings, ninjas, bluespace-anomalies teleporting Syndicate-commandos, giant space fish, giant fucking spiders, swarmer drones, zombies and it is hovering over a planet infested with literal demonic gods... Do you expect security to be a bunch of mall cops?
In what world do new, unrobust, easily scared people play antag? The people that play antag, will not be scared by legs and arm guards, and, if they are... Good. Maybe they'll avoid murderboning and going loud and lethal against security for the 10th round in a row

**Comment by YakumoChen** [comment]
I mean, I kinda expect them to be mall cops until a threat presents itself. Every round is a "fresh" round and you should not be playing basing off past shifts dealing with traitors exploding the station and changelings terrorizing crew.

**Comment by H0gberg** [comment]
The design is one thing to maybe be discussed, yes however it goes without saying, if the antag is smart enough and has two brain cells they will target arms and legs which effectively means sec offs are as armored as your average greytider, it's a very very needed change and it's been stated that the suit designs are mainly coder sprites such as the classic variant, we can make new sprites that are deemed more "lore" friendly as we go along

**Comment by ShadeAware** [comment]
These sprites ARE lore friendly, they don't need changed at all unless you plan to add whatever new sprite as a reskin.

**Comment by Azarak** [comment]
I dont think replacing the Disabler is a good idea. Having the lethal functionality of the gun be enabled by alert level discourages security meetups in the brig and gear distribution. You just get the alert level up and that's it, it takes away from any interactions you could have with warden/HoS, besides, HoS is the one to give you orders anyway. So less interaction with your overseer in person, possibly leading to more vigilantish behaviours, just cause the alert level allows you to, and you didnt meet up with your crew at the brig, because you didn't have to, so you dont have a plan. I'm very against the new rifles

**Comment by CameronWoof** [comment]
> I dont think replacing the Disabler is a good idea. Having the lethal functionality of the gun be enabled by alert level discourages security meetups in the brig and gear distribution. You just get the alert level up and that's it, it takes away from any interactions you could have with warden/HoS, besides, HoS is the one to give you orders anyway. So less interaction with your overseer in person, possibly leading to more vigilantish behaviours, just cause the alert level allows you to, and you didnt meet up with your crew at the brig, because you didn't have to, so you dont have a plan. I'm very against the new rifles

Would you accept them as a research item, instead?

**Comment by H0gberg** [comment]
however at the same time I think it also stops these meetups from becoming, "lets go full lethal riot shotguns on these people who broke a window" from what I've seen when sec tends to all meet up they almost always go WAY too ham when it comes to getting weapons. interaction and communication still happens a LOT via comms and such and sometimes in these situations you just don't have the time to do these meetups, I don't think these guns sacrifice any PR, instead they help enhance the sec experience.

**Comment by Azarak** [comment]
You can't be ready for everything at all times, besides that makes the antagonists and threats have less time to do anything, which puts more pressure on them, possibly denying them of interactions with people or makes them do mistakes which end up in lots of murder. Also I dont think talking on comms is anywhere as good as meeting up and getting a briefing/geared up.

File: code/modules/projectiles/guns/energy/energy_gun.dm [hunk 2]
```diff
@@ -150,3 +150,80 @@
 				add_overlay("[icon_state]_fail_1")
 			if(151 to INFINITY)
 				add_overlay("[icon_state]_fail_2")
+
+/obj/item/gun/energy/e_gun/large //SPECIAL THANKS to Jake Park for helping me with the alert lock code! You're the best, man <3
```
Comment (Azarak) [inline code/modules/projectiles/guns/energy/energy_gun.dm:154]: Remove this gun, or atomize it into its own PR, then I'll be in favor for merging the PR

**Review by Azarak — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/guns/energy/energy_gun.dm (hunk 2 above)
Comment (ShadeAware) [reply code/modules/projectiles/guns/energy/energy_gun.dm:154]: No.

Comment (ShadeAware) [reply code/modules/projectiles/guns/energy/energy_gun.dm:154]: More specifically, No for reason I stated in #Dev-Main on the discord.

**Review by Azarak — APPROVED** [review]

**Comment by Gandalf2k15** [comment]
Okay. full merging.

**Pull request merged**